data/card_catalogue/social.json

```diff
@@ -74,7 +95,10 @@
     "upgrade": "Lie",
     "tier": "blue",
     "details": {
-      "rules": "Target NPC fails their current action and loses focus on you",
+      "rules": {
+        "active": "Double-disadvantage an NPC currently performing an action.",
+        "passive": ""
+      },
```

Comment:
We've talked about this card being busted before, how does double-disadvantage sound in place of instantly failing the current action?

Reply:
I think it's starting to deviate from the flavour of the card.

How about `Tempt an NPC into their compulsion`
`Gregory wanted to pursue the thief, but he couldn't stand leaving the candy wrappers on his spotless kitchen floor`

It fits with the phone picture you made for this card, since looking at your phone is a pretty typical compulsion

Reply:
There's natural variance built into the card since each NPC will have a different compulsion.